roles/openshift_node/tasks/main.yml

```diff
@@ -37,7 +37,8 @@
 - name: Save firewalld port for OpenShift
   firewalld: port=10250/tcp permanent=true state=enabled
 
+  # Always bounce service to pick up new credentials
 - name: Enable OpenShift
-  service: name=openshift-node enabled=yes state=started
+  service: name=openshift-node enabled=yes state=restarted
```

Comment:
We've decided to allow this with a FIXME comment until we fix the larger rsync / kubelet restart ordering issue (double configure steps) in a separate PR.